client/my-sites/patterns/components/pattern-gallery/client.tsx

```diff
@@ -169,7 +169,8 @@ export const PatternGalleryClient: PatternGalleryFC = ( props ) => {
 								'pattern-preview--grid': isGridView,
 								'pattern-preview--list': ! isGridView,
 							} ) }
-							title="…"
+							// Non-breaking space
+							title={ '\u00A0' }
```

Comment:
I went with an empty string instead of the non-breaking there. Not rendering `.pattern-preview__title` didn't seem beneficial, but the `nbsp` wasn't necessary anymore, so it makes sense to remove it.